packages/ia-components/sandbox/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -35,6 +54,7 @@ export default class BookReaderWrapper extends Component {
       searchInsideUrl: '/fulltext/inside.php',
       initialSearchTerm: null,
       imagesBaseURL: '/bookreader/BookReader/images/',
+      //TODO-ISA what is going on here, a comment would be useful ? (same on bookreader-wrapper-main and bookreader-wrapper-jsia)
```

Comment:
Yes, definitely handy.  Will get that in to clarify, or abstract as passable options.

Reply:
this will get taken care of with this ticket: https://github.com/internetarchive/iaux/issues/300